source/developers-guide/plugin-system/index.md

```diff
@@ -275,6 +275,32 @@ As of Shopware 5.2.2 you can also register commands as a service and tag it with
 
 You can read more in the Symfony Documentation: [How to Define Commands as Services](https://symfony.com/doc/2.8/cookbook/console/commands_as_services.html).
 
+### Add menu items to backend
+
+```xml
+<!-- Resources/menu.xml -->
+<?xml version="1.0" encoding="utf-8"?>
+<menu xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="menu.xsd">
```

Comment:
Please use `https://raw.githubusercontent.com/shopware/shopware/5.2/engine/Shopware/Components/Plugin/schema/menu.xsd` instead of `menu.xsd`.